include/seqan3/range/views/repeat.hpp

```diff
@@ -50,6 +50,9 @@ class repeat_view : public std::ranges::view_interface<repeat_view<value_t>>
     //!\brief The sentinel type is set to std::default_sentinel_t.
     using sentinel_type = std::default_sentinel_t;
 
+    //!\brief The sentinel type is set to std::ranges::default_sentinel_t.
```

Comment:
I think this Line should contain something else. Maybe `The single value type is set to XXX`

Reply:
You are absolutely right :)